packages/shared/src/components/auth/AuthenticationBanner.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { ReactElement } from 'react';
+import classed from '../../lib/classed';
+import { OnboardingHeadline } from './OnboardingHeadline';
+import AuthOptions, { AuthDisplay } from './AuthOptions';
+import { AuthTriggers } from '../../lib/auth';
+import { MemberAlready } from '../onboarding/MemberAlready';
+import { useAuthContext } from '../../contexts/AuthContext';
+
+const Container = classed('div', 'flex flex-col w-[23.25rem]');
+
+export function AuthenticationBanner(): ReactElement {
+  const { showLogin } = useAuthContext();
+
+  return (
+    <div
+      className="fixed bottom-0 left-0 z-modal flex w-full flex-row justify-center gap-6 py-10"
+      style={{
+        background: `linear-gradient(270deg, rgba(206, 61, 243, 0.16) 0%, rgba(113, 71, 237, 0.16) 100%), var(--Colors-Background-default, #0E1217)`,
```

Comment:
Can we change the color to pepper 90?
```suggestion
        background: `linear-gradient(270deg, rgba(206, 61, 243, 0.16) 0%, rgba(113, 71, 237, 0.16) 100%), var(--Colors-Background-default, ${colors.pepper['90']})`,
```

Reply:
Ah, good point. Though I think we should use the one that varies based on theme. Will update this quickly.